src/utils/create_label.ts

```diff
@@ -0,0 +1,5 @@
+export function createLabel(name: string, id: string): string {
```

Comment:
- dashed-case (file name)
- Shouldn't this file be placed where the widgets are? (Maybe in a file named `label.ts`)